### DISCUSSION TITLE
text styling

### DISCUSSION THREAD

**Discussion by moolight-seashell** [AppFlowy-IO/AppFlowy#3531] [General help wanted]
there is a way to change h1 h2 h3 colors and size by default ?

like a css h1{color} ?

and there is a way to have more than 3 level of title ?

the doc about styling is very unclear about what item do what

**Comment by annieappflowy** [comment]
@moolight-seashell , you can select the h1 text and apply a different font color:
<img width="744" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/12026239/254900b1-62c5-48c3-8dfb-81d19e360f7f">

We only offer three levels at the moment.

**Comment by moolight-seashell** [comment]
yea i mean apply a global stylesheet to the whole note folder

for example  stylizing all h1 in red so that even new h1s are automatically stylized in the same way.

**Comment by turbodude** [comment]
I would also like to be able to apply some default styles, specifically for the color of body text. Can't wrap my head around the decision (seemingly originated in Notion) of using washed out low-contrast text colors by default and not letting the user to change that.
Appflowy does allow for custom colors, but one would have to apply them to each new text element, which is just not practical.
